docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -26,6 +26,15 @@ in ClusterAPI are kept in sync with the versions used by `sigs.k8s.io/controller
 - github.com/onsi/gomega: v0.17.0 => v0.18.1 (derived from controller-runtime)
 - k8s.io/kubectl: v0.23.5 => 0.24.0
 
+<aside class="note warning">
```

Comment:
Given that this doc is more like a provider migration guide. I'm not sure if we should add the note here. But no strong objections from my side, just feels a bit out of place

Reply:
no bigger opinions on that from my side.

Reply:
no bigger opinions on that from my side. For me it is the link between the affected version to the troubleshooting section.

Reply:
Without any kind of such linkage a user would just fail and then hopefully find the troubleshooting section.

This way users which read the migration guide should get aware of the issue.

Reply:
Yup it's just that this document is the documentation for infra provider authors on how to migrate their infra provider.

So I wouldn't expect end users to find warnings that way. I guess this mostly shows a lack of release notes apart from the auto-generated ones where we can add "custom notes" over time :)

Reply:
🤔 fair. So we should we remove the cross reference then I think?